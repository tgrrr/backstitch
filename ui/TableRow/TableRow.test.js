import React from 'react'
import renderer from 'react-test-renderer'
import { TableRow } from '.'

it('TableRow: default', () => {
    const component = renderer.create(
        <TableRow>
            <>
                TableRow
            </>
        </TableRow>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
