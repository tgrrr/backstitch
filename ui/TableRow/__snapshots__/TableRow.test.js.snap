// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRow: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Tablerow "
    data-testid="Tablerow"
  >
    <tr
      className="MuiTableRow-root"
      role={null}
    >
      TableRow
    </tr>
  </div>
</div>
`;
